subsys/bluetooth/common/Kconfig

```diff
@@ -8,8 +8,16 @@
 
 if BT_HCI
 
+config BT_HCI_VS
+	bool "Zephyr HCI Vendor-Specific Commands"
+	default y
+	help
+	  Enable support for the Zephyr HCI Vendor-Specific Commmands in the
```

Comment:
Would you mind specifying which particular commands are enabled by this vs `VS_EXT`?

Reply:
See my comment above

Reply:
yeap, updated